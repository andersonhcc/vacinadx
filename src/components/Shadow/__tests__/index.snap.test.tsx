/* eslint-disable react/react-in-jsx-scope */
import {Shadow} from '..';
import {View} from 'react-native';
import renderer from 'react-test-renderer';

describe('Shadow', () => {
  it('should render with children', () => {
    const {toJSON} = renderer.create(
      <Shadow>
        <View />
      </Shadow>,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
