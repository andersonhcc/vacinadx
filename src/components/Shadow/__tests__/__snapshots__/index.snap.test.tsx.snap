// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shadow should render with children 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "elevation": 3,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.15,
      "shadowRadius": 2.22,
    }
  }
>
  <View />
</View>
`;
